.github/workflows/hasura-ci-migrations.yaml

```diff
@@ -0,0 +1,38 @@
+name: Hasura Migration
+on:
+  push:
+    branches:
+      - master
+jobs:
+  hasura_migration:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v3
+
+      - name: Apply Hasura Metadata
+        uses: tibotiber/hasura-action@v3.0
+        with:
+          args: metadata apply
+        env:
+          HASURA_ENDPOINT: ${{ secrets.HASURA_ENDPOINT }}
+          HASURA_ADMIN_SECRET: ${{ secrets.HASURA_ADMIN_SECRET }}
+          HASURA_WORKDIR: local-to-cloud
+
+      - name: Apply Hasura Migrations
+        uses: tibotiber/hasura-action@v3.0
```

Comment:
Who is tibotiber? Can we move this action to Hasura org? (Not super critical but if it's quick we should do it).